src/valueUpdatedMethod.js

```diff
@@ -0,0 +1,7 @@
+export default [
```

Comment:
You need change file name from `valueUpdatedMethod.js` to `valueUpdateMethod.js`.